R/delineate.R

```diff
@@ -28,7 +28,7 @@
 #' @export
 delineate_corridor <- function(
   city_name, river_name, crs = NULL, bbox_buffer = NULL,
-  initial_method = "buffer", initial_buffer = 1000, dem = NULL,
+  initial_method = "valley", initial_buffer = 1000, dem = NULL,
```

Comment:
Do we need to set `initial_buffer` in this case? Or would it be better to set it to NULL for clarity in the default parameters?